tests/infra/jwt_issuer.py

```diff
@@ -0,0 +1,190 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache 2.0 License.
+import infra.crypto
+import base64
+from http.server import HTTPServer, BaseHTTPRequestHandler
+from http import HTTPStatus
+import ssl
+import threading
+from contextlib import AbstractContextManager
+import tempfile
+import json
+import time
+from loguru import logger as LOG
+
+
+def make_authn_bearer_header(jwt):
```

Comment:
This should be `authz`, not `authn`, since its setting the `authorization` header? Or perhaps just `make_bearer_header`.

Reply:
Good point! Changed it `make_bearer_header`.